Source/WebCore/accessibility/AXObjectCache.h

```diff
@@ -397,6 +397,10 @@ class AXObjectCache final : public CanMakeWeakPtr<AXObjectCache>, public CanMake
     void onSlottedContentChange(const HTMLSlotElement&);
     void onStyleChange(Element&, OptionSet<Style::Change>, const RenderStyle* oldStyle, const RenderStyle* newStyle);
     void onStyleChange(RenderText&, StyleDifference, const RenderStyle* oldStyle, const RenderStyle& newStyle);
+#if ENABLE(ACCESSIBILITY_ISOLATED_TREE)
+    bool onFontChange(Element&, const RenderStyle*, const RenderStyle*);
+    bool onTextColorChange(Element&, const RenderStyle*, const RenderStyle*);
```

Comment:
Can we add a comment to each of these explaining what the returned `bool` value means?